### ISSUE TITLE
Review vessel uri definition, allow for uuid, mmsi, url

### ISSUE THREAD

**Issue by rob42** [SignalK/signalk-server-java#45]
See https://github.com/SignalK/freeboard-sk/pull/22
Server needs to honor the mmsi (and url) names same as uuid

**Comment by sbender9** [comment]
Yeah, I'm a bit confused now. Seems like things are not how they should be with the servers and the spec seems inconsistent.

`/self/mmsi` should obviously return the mmsi: `1233455`

So then shouldn't `/self/uuid` be just the uuid: `b7590868-1d62-47d9-989c-32321b349fb9`?
